src/main/java/frc/robot/commands/autonomous/SplooshAndVamooseEdge.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands.autonomous;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.commands.*;
+import frc.robot.subsystems.ArmFirstJoint;
+import frc.robot.subsystems.ArmSecondJoint;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Gripper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SplooshAndVamooseEdge extends BasePathAuto {
```

Comment:
I'm pretty sure this didn't work, so you can delete this class?

Reply:
Uhhhh I wouldn't risk it